shared/src/business/utilities/DateHandler.test.js

```diff
@@ -521,4 +521,16 @@ describe('DateHandler', () => {
       expect(validDate).toEqual('2001-01-01T05:00:00.000Z');
     });
   });
+
+  describe('subtractISODates)', () => {
+    it('return an ISO string of the previous year', () => {
```

Comment:
Do we need a case that takes timezones that use DST into consideration?

Ex: subtracting 1 month from 2AM on 4/13/22 (ISO date "2022-04-13T02:00:00.000-0400") should return 3AM on 3/13/22 (ISO date "2022-03-13T03:00:00.000-0400") because there is no 2AM on 3/13/22.